CONTRIBUTING.md

```diff
@@ -48,7 +49,7 @@ You do not have to have finished your feature or bug fix before submitting a PR;
 
 Before submitting PRs:
 
-- make sure your code still conforms to the formatting standard (using Black formatter: `$ black -t py36 .`)
+- make sure that your code passes code checks: `$ pre-commit run --all-files`
```

Comment:
In which case we probably don't need lines like this?

Reply:
Sure, I was doubting about include it

Reply:
Sure, I was doubting about to include it